Tools/autotest/param_metadata/xmlemit_mp.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+from xml.sax.saxutils import escape, quoteattr
+
+from emit import Emit
+from param import known_param_fields, known_units
+
+
+# Emit APM documentation in an machine readable XML format for Mission Planner
+class XmlEmitMP(Emit):
+    def __init__(self):
+        Emit.__init__(self)
+        wiki_fname = 'parametermetadata.xml'
+        self.f = open(wiki_fname, mode='w')
```

Comment:
Shouldn't this be mp_fname ?
```suggestion
        mp_fname = 'parametermetadata.xml'
        self.f = open(mp_fname, mode='w')
```

Reply:
done